pubspec.yaml

```diff
@@ -7,7 +7,7 @@ description: >-
 repository: https://github.com/dart-lang/intl_translation
 
 environment:
-  sdk: '>=2.12.0 <3.0.0'
+  sdk: '>=2.17.0 <3.0.0'
```

Comment:
Is moving this up to `2.17.0` intentional? The current stable version is `2.18.1`; it's nice (but not critical) to support a few older stable versions as well.

Whatever min. version we have here we should also reflect in the CI config as well (`.github/workflows/build.yaml`) - i.e., test against the min sdk we've specified, as well as the current stable sdk.

Reply:
I assumed that I should pick the lowest version which still "works", i.e. has the features we need in this package. In this case, I chose 2.17 to use the `super.parameterName` shorthand syntax. I don't mind switching to 2.18.1 though.

Reply:
> I assumed that I should pick the lowest version which still "works"

Yup for sure. That's what I was trying to say above - that we should prefer (all other things being equal) to support a few major sdk versions.

It sounds like you've moved it forward to 2.17.0 to use a new sdk feature - sgtm. That would have us supporting dart 2.17 and 2.18.